pkg/controller/middleware/email_verified_test.go

```diff
@@ -54,87 +54,79 @@ func TestRequireEmailVerified(t *testing.T) {
 		prompted      bool
 		membership    *database.Membership
 		code          int
-	}{
-		{
-			name:       "missing_membership",
-			membership: nil,
-			code:       400,
-		},
-		{
-			name:          "optional_verified",
-			emailVerified: true,
-			membership: &database.Membership{
-				Realm: &database.Realm{
-					EmailVerifiedMode: database.MFAOptional,
-				},
+	}{{
```

Comment:
The previous format is better for avoiding unnecessary diffs

Reply:
agree on this - this style of literal slice is all over the codebases, let's stay consistent

Reply:
Sure...